crates/holochain_zome_types/src/warrant.rs

```diff
@@ -142,6 +164,8 @@ impl WarrantProof {
     /// Basis hash where this warrant should be delivered.
     /// Warrants always have the authoring agent as a basis, so that warrants
     /// can be accumulated by the agent activity authorities.
+    // TODO: It seems more logical that the basis is the warrantee's pub key.
+    // Then the warrantee's agent authority holds the warrants.
```

Comment:
Is that not what's happening here? It's the author of the action that got warranted?

Reply:
That's from before I made the decision to make this change on this PR.

Reply:
removed https://github.com/holochain/holochain/pull/4937/commits/8ed5314d7dead651451c42b157c3aef0e17915d0